client/views/components/genny/genny-table/GennyTable.jsx

```diff
@@ -231,7 +231,8 @@ class GennyTable extends Component {
                                 'Cell': cellInfo => {
                                     const cell = cellInfo.row[ ( attribute && attribute.attributeCode ) || attributeCode];
                                     const value = cell && cell.value;
-                                    const dataType = ( cell && cell.type ) || BaseEntityQuery.getAttribute(( attribute && attribute.attributeCode ) || attributeCode).dataType.typeName ;
+                                    const attributeBE = BaseEntityQuery.getAttribute( attributeCode );
```

Comment:
reason for this change? should be looking for `attribute.attributeCode` with `attributeCode` as a fallback

Reply:
that was probably not me, when i do git diff i can't see that i did this . someone probably directly pushed to dev.